File: SAP_APIv1/SAP_API/DTOs/Requests/AvailableProgramsResponse.cs
using SAP_API.DTOs.Responses;
using System.Collections.Generic;

namespace SAP_API.DTOs.Requests
{
    public class AvailableProgramsResponse
    {
        public List<AvailableBakingProgramResponse> BakingPrograms { get; set; }
        public bool AllProductsCanBeSuccessfullyArranged { get; set; }
        public bool IsThereEnoughStockedProducts { get; set; }
    }
}

File: bakery-app/src/models/Requests/AvailableProgramsResponse.ts
import AvailableBakingProgramResponse from "../Responses/AvailableBakingProgramResponse";

export default interface AvailableProgramsResponse {
    bakingPrograms: AvailableBakingProgramResponse[];
    allProductsCanBeSuccessfullyArranged: boolean;
    isThereEnoughStockedProducts: boolean;
}
